Source/WebCore/page/FocusController.cpp

```diff
@@ -1222,14 +1230,18 @@ bool FocusController::advanceFocusDirectionallyInContainer(Node* container, cons
 
 bool FocusController::advanceFocusDirectionally(FocusDirection direction, KeyboardEvent* event)
 {
-    Document* focusedDocument = focusedOrMainFrame().document();
+    RefPtr focusedOrMainFrame = this->focusedOrMainFrame();
+    if (!focusedOrMainFrame)
+        return false;
+
+    RefPtr focusedDocument = focusedOrMainFrame->document();
     if (!focusedDocument)
         return false;
 
     focusedDocument->updateLayoutIgnorePendingStylesheets();
 
     // Figure out the starting rect.
-    Node* container = focusedDocument;
+    RefPtr<Node> container = focusedDocument.get();
```

Comment:
I believe `.get()` is unnecessary

Reply:
While you're correct, the style bot complains if it's removed.  Since it's equivalent, I'm going to leave it.

Reply:
Weird decision..

Reply:
Let's make write more unnecessary code because of a style bot false positive?